### PR TITLE
Updated all QuestionType icons and labels for clarity

### LINKED ISSUE #1904: Change icon for "Text"
The icon for/next to "Text" is unexpected and not standard. Change icon to the standard capital T for text.

### PR DESCRIPTION
**Pull request by th174**
### Description
This PR updates all the question type icons and labels in the Create New Question menu, in accordance with this proposal from @rahaghas: https://github.com/seattle-uat/civiform/issues/1904#issuecomment-1114249486

This PR doesn't remove the static question type as that would be out of scope.

### Checklist
- [ ] Created tests which fail without the change (if possible)
- [ ] Extended the README / documentation, if necessary

### Screenshot
image

### Issue(s)
Fixes #1904

### CODE REVIEW

**Review by goldblatt — COMMENTED** [review]
what happened to the attr that were added here?

**Comment by th174** [comment]
> what happened to the attr that were added here?

The are no longer needed to create the svg icons, using the new svg paths.

**Review by goldblatt — APPROVED** [review]

**Comment by bion** [comment]
🤩

**Comment by shanemc-goog** [comment]
Per the discussion on #1904 I think we want to keep the $ icon for currency

File: server/app/views/components/Icons.java [hunk 1]
```diff
@@ -3,62 +3,106 @@
...
+  // Check Box
+  public static final String CHECKBOX_SVG_PATH =
+      "M19 3H5c-1.1 0-2 .9-2 2v14c0 1.1.9 2 2 2h14c1.1 0 2-.9 2-2V5c0-1.1-.9-2-2-2zm0"
+          + " 16H5V5h14v14zM17.99 9l-1.41-1.42-6.59 6.59-2.58-2.57-1.42 1.41 4 3.99z";
+  // Payments
   public static final String CURRENCY_SVG_PATH =
-      "M12 8c-1.657 0-3 .895-3 2s1.343 2 3 2 3 .895 3"
-          + " 2-1.343 2-3 2m0-8c1.11 0 2.08.402 2.599 1M12 8V7m0 1v8m0 0v1m0-1c-1.11"
-          + " 0-2.08-.402-2.599-1M21 12a9 9 0 11-18 0 9 9 0 0118 0z";
+      "M19,14V6c0-1.1-0.9-2-2-2H3C1.9,4,1,4.9,1,6v8c0,1.1,0.9,2,2,2h14C18.1,16,19,15.1,19,14z"
```
Comment (shanemc-goog) [inline server/app/views/components/Icons.java:24]: Per the discussion on https://github.com/seattle-uat/civiform/issues/1904 I think we want to keep the $ icon for currency

**Review by shanemc-goog — CHANGES_REQUESTED** [review]

File: server/app/views/components/Icons.java (hunk 1 above)
Comment (th174) [reply server/app/views/components/Icons.java:24]: Sure sounds good.

**Review by shanemc-goog — APPROVED** [review]

**Pull request merged**